src/outfit.ts

```diff
@@ -94,6 +94,14 @@ export function freeFightOutfit(requirement?: Requirement): void {
   if (haveEquipped($item`Buddy Bjorn`)) bjornifyFamiliar(bjornChoice.familiar);
   if (haveEquipped($item`Crown of Thrones`)) enthroneFamiliar(bjornChoice.familiar);
   if (haveEquipped($item`Snow Suit`) && get("snowsuit") !== "nose") cliExecute("snowsuit nose");
+
+  if (
+    (finalRequirement.maximizeOptions.forceEquip ?? []).some(
+      (equipment) => !haveEquipped(equipment)
+    )
+  ) {
+    throw new Error("Failed to don outfit appropriately!");
```

Comment:
Maybe the error could be a little more explicit & helpful?
`Maximizer failed to equip desired equipment. Maybe try 'refresh all' and run again?`